src/core/annotations/field-value.ts

```diff
@@ -0,0 +1,38 @@
+import {isPresent} from 'angular2/src/facade/lang';
+
+
+declare var Symbol: any;
```

Comment:
No. We simply declare the ambient variable. Otherwise it's not defined and TypeScript will scream. Apparently `es6-shim` doesn't have that.